editor/filesystem_dock.cpp

```diff
@@ -1890,22 +1892,15 @@ void FileSystemDock::_move_operation_confirm(const String &p_to_path, bool p_cop
 		if (p_overwrite == OVERWRITE_RENAME) {
 			new_paths.write[i] = _get_unique_name(to_move[i], p_to_path);
 		} else {
-			new_paths.write[i] = p_to_path.path_join(to_move[i].path.get_file());
+			new_paths.write[i] = p_to_path.path_join(to_move[i].is_file ? to_move[i].path.get_file() : to_move[i].path.left(-1).get_file());
```

Comment:
```suggestion
			new_paths.write[i] = p_to_path.path_join(to_move[i].to_move[i].trim_suffix("/").get_file());
```
Simpler and does the same.